src-vue/src/components/PlayButton.vue

```diff
@@ -13,7 +13,7 @@ export default defineComponent({
 
             switch(this.$store.state.northstar_state) {
                 case NorthstarState.GAME_NOT_FOUND:
-                    return "Titanfall2 not found";
+                    return "Select game folder";
```

Comment:
```suggestion
                    return "Select Titanfall2 game folder";
```

Might be too long but would make it more obvious what the button does ^^